spec/controllers/gardens_controller_spec.rb

```diff
@@ -12,11 +12,72 @@
 
 require 'rails_helper'
 
-describe GardensController do
-  login_member
+RSpec.describe GardensController, type: :controller do
+  include Devise::Test::ControllerHelpers
+  let(:valid_params) { { name: 'My second Garden' } }
 
-  def valid_attributes
-    member = FactoryGirl.create(:member)
-    { name: 'My Garden', owner_id: member.id }
+  context "when not signed in" do
+    let(:garden) { double('garden') }
+    describe 'GET new' do
+      before { get :new, id: garden.to_param }
```

Comment:
Right - I was confused because you were using `before` in cases where there was only one assertion.